common.eai

```diff
@@ -2031,7 +2031,10 @@ function ApplyChatVars takes string s_in returns string
     exitwhen i >= l
     set c = GetChar(s_in, i)
     if c == "" then
-      return s_in // Means a character not in the StandardCharTable.txt so return original text as we cannot perform placeholder replacement.
+      set key = null
```

Comment:
This is unnecessary, strings do not leak, just like other primitives.

Reply:
Oh, I found some claims that strings may indeed leak, so just consider it my bad habit